### PR TITLE
Adjustment to prevent menu text wrapping

### PR DESCRIPTION
**Pull request by timarney**
@smcmurtry please give this a test.

### CODE REVIEW

File: src/screens/menu/components/MenuButton.tsx [hunk 1]
```diff
@@ -33,4 +35,8 @@ const styles = StyleSheet.create({
     justifyContent: 'space-between',
     alignItems: 'center',
   },
+  menuBoxWrap: {
+    minWidth: 200,
```
Comment (smcmurtry) [inline src/screens/menu/components/MenuButton.tsx:39]: Looks like this is the change that actually fixes the issue. I reverted all the other changes and introduced this one and that fixed it.

File: src/screens/menu/components/MenuButton.tsx [hunk 2]
```diff
@@ -33,4 +35,8 @@ const styles = StyleSheet.create({
...
+    flexWrap: 'nowrap',
```
Comment (smcmurtry) [inline src/screens/menu/components/MenuButton.tsx:40]: I don't think this is doing anything - at least on Android.

**Review by smcmurtry — APPROVED** [review]
Fixed! This is on my MotoG4 with window width of 360:

Screen Shot 2021-05-11 at 8 56 54 AM

File: src/screens/menu/components/MenuButton.tsx (hunk 2 above)
Comment (timarney) [reply src/screens/menu/components/MenuButton.tsx]: Removed the flexWrap: 'nowrap',

**Pull request merged**